### DISCUSSION TITLE
Welcome to Practice-Contribution Discussions!

### DISCUSSION THREAD

**Discussion by Evavic44** [CommunityPro/Practice-Contribution#1] [Ideas]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Mannuel25** [comment]
Hey Victor, what else is to be added to the repo. Assign one or two tasks to me.

**Reply by Evavic44** [reply]
Okay, I'm creating issues to be handled now. Stay tuned.

**Reply by Evavic44** [reply]
Assigned one issue #4
